scripts/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: "3"
+
+services:
+    rabbitmq:
+        image: rabbitmq:${RABBITMQ_VERSION:-3.8}-management-alpine
+        ports:
+            - 15672:15672
```

Comment:
To make things more explicit, could you also pass the environment variables that you defined for the username, password in here?

```
env:
  RABBITMQ_DEFAULT_USER: ${RABBITMQ_USERNAME:guest}
  RABBITMQ_DEFAULT_PASS: ${RABBITMQ_PASSWORD:guest}
```